ament_xmllint/ament_xmllint/main.py

```diff
@@ -39,15 +39,21 @@ def main(argv=sys.argv[1:]):
         nargs='*',
         default=[os.curdir],
         help='The files or directories to check. For directories files ending '
```

Comment:
```suggestion
        help='The files or directories to check. For directories, only files ending '
```

(I know you didn't add this, but I think we can improve the wording a bit while we are here)